.vscode/launch.json

```diff
@@ -8,7 +8,7 @@
       "name": "Launch CSTG (Chrome)",
       "type": "chrome",
       "request": "launch",
-      "url": "http://localhost:9090",
+      "url": "http://localhost:9190",
```

Comment:
Is this just to move it off the commonly used port of 9090?

Reply:
Should the .vscode directory be checked in to git here?  isn't that pretty specific for an open source project?

Reply:
These are all the launch settings - without it you can't launch the tests. I'd normally have used npm tasks but someone else set it up this way, and I quite like it actually. I don't think it's worth moving it all to npm, there's a fair bit of functionality in here. I think this is worth checking in.

And yeah, it's to move away from a commonly-used port.